plugins/onepad/SDL/joystick.h

```diff
@@ -56,4 +56,5 @@ class JoystickInfo : GamePad
     SDL_GameController *m_controller;
     SDL_Haptic *m_haptic;
     std::array<int, NB_EFFECT> m_effects_id;
+    bool m_no_error;
```

Comment:
Oh I might scew up a bit in my rebase. I will check it at home.

You can take more time.